app/js/components/orderEntry/OrderEntryPage.jsx

```diff
@@ -78,6 +85,32 @@ export class OrderEntryPage extends React.Component {
       );
     }
 
+    if (dateFormat === null || dateError) {
+      return (
+        <div className="error-notice">
+          <p>
+            Configuration for <strong>orderentryowa.dateAndTimeFormat</strong> {dateError === 'incomplete config' ? 'is incomplete' : 'does not exist'}.
+            Please contact your administrator for more information.
+          </p>
+          <p>
+            As an Administrator,&nbsp;
+            {
+              dateError === 'incomplete config' ?
+                <span>
+                  please ensure that you have created a valid <strong>date and time format</strong>.
+                </span> :
+                <span>
+                  ensure that you have created a setting called
+                  <strong>orderentryowa.dateAndTimeFormat</strong>
+                  &nbsp;
+                  with a value corresponding to DD-MM-YYYY HH:mm.
```

Comment:
Why do you hard code the above date format?

Reply:
It's not reflective as a date, it's a sample of what the value is supposed to be, when the user is setting the date and time format in the global properties

Reply:
Then the message needs to change and reflect so.

Reply:
@dkayiwa I've made changes, please take a look